src/io/sampling.jl

```diff
@@ -41,7 +44,18 @@ function adjust_samples(spec::DataFrame, df::DataFrame)::DataFrame
 
     # Same for fogging/shading
     not_fogged = (df.fogging .== 0) .& (df.SRM .== 0)
-    df[not_fogged, contains.(names(df), "shade_")] .= 0.0
+    df[not_fogged, contains.(names(df), "fog_") .& contains.(names(df), "shade_")] .= 0.0
+
+    # Normalize MCDA weights for fogging scenarios
+    guided_fogged = (df.fogging .> 0.0) .& (df.guided .> 0)
+    df[guided_fogged, weights_fog_crit.fieldname] .= mcda_normalize(
+        df[guided_fogged, weights_fog_crit.fieldname],
+    )
+    # Normalize MCDA weights for seeding scenarios
+    guided_seeded = .!(not_seeded) .& is_guided
```

Comment:
`is_guided` does not exist any more.

Tests fail due to this.

Reply:
Fixed now, sorry, and tests are passing on my end